ru/mail/chrome/messenger/addressbook/abMainWindow.dtd

```diff
@@ -28,51 +28,51 @@
 <!ENTITY closeCmd.accesskey                             "C">
 <!ENTITY printSetupCmd.label                            "Параметры страницы…">
 <!ENTITY printSetupCmd.accesskey                        "u">
-<!ENTITY printPreviewCardViewCmd.label                  "Print Preview Card">
+<!ENTITY printPreviewCardViewCmd.label                  "Предварительный просмотр карточки">
 <!ENTITY printPreviewCardViewCmd.accesskey              "v">
 <!ENTITY printPreviewAddressBook.label                  "Предварительный просмотр адресной книги">
 <!ENTITY printPreviewAddressBook.accesskey              "B">
-<!ENTITY printCardViewCmd.label                         "Print Card...">
-<!ENTITY printCardViewCmd.accesskey                     "P">
+<!ENTITY printCardViewCmd.label                         "Печать карточки…">
+<!ENTITY printCardViewCmd.accesskey                     "p">
 <!-- LOCALIZATION NOTE (printCardViewCmd.key) : DONT_TRANSLATE -->
 <!ENTITY printCardViewCmd.key                           "P">
 <!ENTITY printAddressBook.label                         "Печать адресной книги…">
 <!ENTITY printAddressBook.accesskey                     "A">
 
 <!ENTITY quitApplicationCmdWin.label                    "Выход"> 
 <!ENTITY quitApplicationCmdWin.accesskey                "x">
-<!ENTITY quitApplicationCmd.label                       "Выход"> 
+<!ENTITY quitApplicationCmd.label                       "Завершить"> 
 <!ENTITY quitApplicationCmd.accesskey                   "Q">
 <!ENTITY quitApplicationCmd.key "Q">
 
 <!-- Edit Menu -->
 <!ENTITY editMenu.label                                 "Правка">
-<!ENTITY editMenu.accesskey                             "E">
+<!ENTITY editMenu.accesskey                             "e">
 <!ENTITY undoCmd.label                                  "Отменить">
 <!ENTITY undoCmd.key                                    "Z">
-<!ENTITY undoCmd.accesskey                              "U">
+<!ENTITY undoCmd.accesskey                              "u">
 <!ENTITY redoCmd.label                                  "Повторить">
 <!ENTITY redoCmd.key                                    "Y">
-<!ENTITY redoCmd.accesskey                              "R">
+<!ENTITY redoCmd.accesskey                              "r">
 <!ENTITY cutCmd.label                                   "Вырезать">
 <!ENTITY cutCmd.key                                     "X">
 <!ENTITY cutCmd.accesskey                               "t">
 <!ENTITY copyCmd.label                                  "Копировать">
 <!ENTITY copyCmd.key                                    "C">
-<!ENTITY copyCmd.accesskey                              "C">
+<!ENTITY copyCmd.accesskey                              "c">
 <!ENTITY pasteCmd.label                                 "Вставить">
 <!ENTITY pasteCmd.key                                   "V">
-<!ENTITY pasteCmd.accesskey                             "P">
+<!ENTITY pasteCmd.accesskey                             "p">
```

Comment:
OK, sorry. I have reverted this hotkey change. Should I commit the fixed file again?